web/README.md

```diff
@@ -0,0 +1,64 @@
+## Getting started
+
+- Download [Hugo](https://gohugo.io/#action) and put it in your `PATH`
+- Install Node packages with `npm install` (or via `yarn`), *OPTIONAL* for SCSS and Javascript
```

Comment:
This file is updated if you run `yarn upgrade`. I will do it when it is necessary (e.g. to get a new version of vtk.js).